src/Navbar.jsx

```diff
@@ -3,14 +3,19 @@ import styled from "styled-components";
 import { Link } from 'react-router-dom';
 
 export const StyledNavbar = styled.nav`
-  text-align: center;
-  background: papayawhip;
-`;
+  background: #ccb7bb;
+  width: 100%;
+  height: 30px;
+  `;
+
+export const StyledLink = styled(Link)`
```

Comment:
May want to position this witihin the navbar, usually home buttons are positioned to the left